.github/workflows/generate-import.yml

```diff
@@ -20,28 +18,31 @@ jobs:
       contents: write
 
     steps:
+      - uses: google-github-actions/release-please-action@v3
+        id: release
+        with:
+          release-type: simple
+
```

Comment:
release-please is supposed to run on every commit, not with a schedule.

Adding release please here has absolutely no benefit, if we're going to use a release, I'd prefer if it were a rolling tag/release

Reply:
You could manually create a release called `import-styles`, and then use `gh release upload --clobber`, which overwrites existing assets of the same name. (upload the same CSS to the same release every 24h)

Reply:
That makes sense hammy suggested we used build artifacts instead. I am unsure where to stand on this myself.
https://github.com/catppuccin/jetbrains/blob/27949117de78e8f33f1d5bbeaec975ac9a7c15fe/.github/workflows/build.yml#L54-L58

Reply:
I mentioned artifacts, they change their URL with every commit, since the action run ID is part of the URL.

Reply:
I mentioned build artifacts as an alternative to avoid the weird actions stuff you were posting on discord 😂 @isabelroses 

A rolling release would be fine, the action would just need to update the assets of the latest release.